parts/linux/cloud-init/artifacts/components.json

```diff
@@ -466,22 +466,22 @@
         {
           "renovateTag": "<DO_NOT_UPDATE>",
           "latestVersion": "v1.28.13-windows-hpc",
-          "previousLatestVersion": "v1.28.13"
```

Comment:
It seems we have a trick, by use latestVersion to cache windows-hpc and previousLatestVersion for non-widnows-hpc, which means both image should have the same version

Reply:
We should probably create a different object in the array for the windows-hpc ones - that way the prevLatestVersion win't be overwritten by the latestVersion. I'll make a PR to seperate them.

Reply:
PR made: windows-hpc

Reply:
Great to see both are supported together. Thanks @timmy-wright to make it working!